src/data/accessibility_de.json

```diff
@@ -16,6 +22,16 @@
             "Als öffentliche Stelle im Sinne der Richtlinie (EU) 2016/2102 sind wir bemüht, unsere Websites und mobilen Anwendungen im Einklang mit den Bestimmungen des Behindertengleichstellungsgesetzes des Bundes (BGG) sowie der Barrierefreien-Informationstechnik-Verordnung (BITV 2.0) zur Umsetzung der Richtlinie (EU) 2016/2102 barrierefrei zugänglich zu machen."
         ]
     },
+    "section-website-compatibility-level": {
+        "title": "Stand der Vereinbarkeit mit den Anforderungen",
+        "textblock": [
+            "<b>Stand der Vereinbarkeit mit den Anforderungen</b>",
+            "Diese Webseite ist wegen der folgenden Ausnahmen teilweise nicht mit dem BGG und der BITV 2.0 vereinbar.",
+            "<b>Nicht barrierefreie Inhalte auf der Webseite www.coronawarn.app</b>",
+            "<p>Die nachstehend aufgeführten Inhalte sind wegen Unvereinbarkeit mit dem BGG und der BITV 2.0 nicht barrierefrei. Wir arbeiten an der baldmöglichen Beseitigung:</p>",
+            "<ul> <li>Die interaktiven Diagramme sind noch nicht in vollem Umfang für Menschen mit Sehbeeinträchtigungen zugänglich.</li> <li>Farbfehlsichtigkeit kann sich negativ auf die Lesbarkeit der Diagramme auswirken.</li> <li>Ältere PDF-Dateien der Nutzungsbedingungen und Datenschutzerklärung sind ggf. nicht barrierefrei. Später eingestellte PDF-Dateien werden in barrierefreier Version erstellt beziehungsweise schnellstmöglich durch eine solche ersetzt.</li> <li>PDF-Dateien. Diese enthalten unvollständige Alternativtexte. Die Tastaturbedienbarkeit ist nicht vollständig gegeben. Zum Teil sind Kontrastfehler enthalten.</li> <li>Benutzerdefinierte Einstellungen: Farben (z. B. Dark Mode), Kontraste und Fokus Cursor werden nicht korrekt unterstützt.</li></ul>"
```

Comment:
```suggestion
            "<ul> <li>Die interaktiven Diagramme sind noch nicht in vollem Umfang für Menschen mit Sehbeeinträchtigungen zugänglich.</li> <li>Farbfehlsichtigkeit kann sich negativ auf die Lesbarkeit der Diagramme auswirken.</li> <li>Ältere PDF-Dateien der Nutzungsbedingungen und Datenschutzerklärung sind ggf. nicht barrierefrei. Später eingestellte PDF-Dateien werden in barrierefreier Version erstellt beziehungsweise schnellstmöglich durch eine solche ersetzt.</li> <li>PDF-Dateien. Diese enthalten unvollständige Alternativtexte. Die Tastaturbedienbarkeit ist nicht vollständig gegeben. Zum Teil sind Kontrastfehler enthalten.</li> <li>Benutzerdefinierte Einstellungen: Farben (z.&nbsp;B. Dark Mode), Kontraste und Fokus Cursor werden nicht korrekt unterstützt.</li></ul>"
```
to ensure that `z. B.` is not broken over two lines

Reply:
Thanks for your suggestion! It has been committed